### DISCUSSION TITLE
NAS-OMV容器配置

### DISCUSSION THREAD

**Discussion by caibingcheng** [caibingcheng/comment#299] [Announcements]
https://imcbc.cn/202209/nas-omv-reprocesss/?code=6d72e8b429524048b85b

**Comment by penndu** [comment]
Unraid了解下！

**Comment by caibingcheng** [comment]
> @penndu
> Unraid了解下！

可以！又了解到了一个新系统！看着社区也很活跃，先收藏了！

**Comment by penndu** [comment]
> @caibingcheng
> > @penndu
> > Unraid了解下！
> 
> 可以！又了解到了一个新系统！看着社区也很活跃，先收藏了！

不过好像是收藏的！

**Comment by penndu** [comment]
打错字了，想说是收费的！
